tools/sof-kernel-dump.sh

```diff
@@ -8,11 +8,20 @@ last_order=$[ $last_order + 1 ]
 log_file=${2:-"/var/log/kern.log"}
 # check whether target file exists
 [[ ! -f "$log_file" ]] && echo "$0 parameter 2 $log_file does not exist" && builtin exit 1
+# load help script
+which sof-get-kernel-line.sh 2>&1 1>/dev/null
```

Comment:
can `which` find the script if it is not in PATH?

Reply:
no,  `which - locate a command`

Reply:
which returns the pathnames of the files (or links) which would be exe‐
       cuted in the current environment, had its arguments been given as  com‐
       mands  in a strictly POSIX-conformant shell.  ***It does this by searching
       the PATH for executable files matching the names of the  arguments.  It
       does not canonicalize path names.***
this is from `man which`, I am afraid you can not locate sof-get-kernel-line.sh, because it only search folders in PATH.